npm/qsharp/ux/circuit.tsx

```diff
@@ -78,41 +180,29 @@ export function CircuitPanel(props: CircuitProps) {
   return (
     <div class="qs-circuit-panel">
       <div>
-        <h1>{props.title}</h1>
+        <h1>
+          {props.title} {props.simulated ? "(Trace)" : ""}
+        </h1>
       </div>
-      {props.circuit ? <Circuit circuit={props.circuit}></Circuit> : null}
       <div class="qs-circuit-error">{error}</div>
       <p>{props.targetProfile}</p>
-      {props.simulating ? (
-        <p>
-          This circuit diagram was generated while running the program in the
-          simulator.
-          <br />
-          <br />
-          If your program contains behavior that is conditional on a qubit
-          measurement result, note that this circuit only shows the outcome that
-          was encountered during this simulation. Running the program again may
-          result in a different circuit being generated.
-        </p>
-      ) : null}
-      {
-        // show tip when the circuit is empty and we didn't run under the simulator (i.e. debugging)
-        !props.simulating &&
-        !props.errorHtml &&
-        props.circuit?.qubits.length === 0 ? (
-          <p>
-            <em>
-              Tip: you can generate a circuit diagram for any operation that
-              takes qubits or arrays of qubits as input.
-            </em>
-          </p>
-        ) : null
-      }
       <p>
-        <a href="https://github.com/microsoft/qsharp/wiki/Circuit-Diagrams-from-Q%23-Code">
-          Learn more
-        </a>
+        {
+          props.simulated
+            ? "WARNING: This diagram shows the result of tracing a dynamic circuit, and may change from run to run."
+            : "\xa0" /* nbsp to keep line height consistent */
```

Comment:
Does this behave differently from using the common `&nbsp;` in HTML?

Reply:
Well, I really wanted to put `&nbsp;` here, but evaluated JS strings get HTML-escaped in JSX elements 😢  (i.e. the user literally sees "&nbsp;" )

This does the same job, but looks clunky to me:

```jsx
{props.simulated ? (
  <p>
    WARNING: This diagram shows the result of tracing a dynamic circuit,
    and may change from run to run.{" "}
  </p>
) : (
  <p>&nbsp;</p> //  to keep line height consistent
)}
```